bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx142.html

```diff
@@ -58,16 +58,11 @@ <h3 class="wnp-subtitle">
               <h3 class="wnp-subtitle">
                 <a href="https://support.mozilla.org/en-US/kb/customize-items-on-firefox-new-tab-page#w_change-whats-on-your-new-tab-page" data-cta-text="Learn more - new tab customization" aria-describedby="desc-new-tab">Make your New Tab feel like home</a>
               </h3>
-                <p class="wnp-body">Create a space that's uniquely yours. Set the mood with Firefox wallpapers, custom colors, or your own photo. Open a new tab and click the <svg class="wnp-inline-icon" width="16" height="16" viewBox="0 0 39 39" fill="currentColor" aria-label="pencil" role="img">
-                    <path d="M 14.102 35.002 C 13.98 35.123 13.82 35.197 13.648 35.212 L 3.672 25.23 C 3.686 25.058 3.761 24.897 3.882 24.775 L 20.409 8.25 C 20.694 7.966 21.155 7.966 21.439 8.25 L 30.632 17.444 C 30.916 17.729 30.916 18.189 30.632 18.474 L 14.107 34.997 L 14.102 35.001 L 14.102 35.002 Z" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" fill="none" stroke-width="2.5" />
-                    <path d="M 26.997 1.659 C 27.852 0.805 29.236 0.805 30.09 1.659 L 37.22 8.789 C 38.074 9.643 38.074 11.029 37.22 11.882 L 31.141 17.962 L 20.918 7.738 L 26.998 1.658 L 26.997 1.659 Z" fill="none" stroke="currentColor" stroke-width="2.5" />
-                    <path d="M 23.27 15.609 L 13.36 25.519" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2.5" />
-                    <path d="M 13.648 35.215 L 1.936 37.738 C 1.486 37.776 1.109 37.398 1.15 36.948 L 3.673 25.232" fill="none" stroke-width="2.5" />
-                    <path d="M 13.648 35.215 L 1.936 37.738 C 1.486 37.776 1.109 37.398 1.15 36.948 L 3.673 25.232" fill="none" stroke="currentColor" stroke-width="2.5" />
-                    <path d="M 6.194 36.811 L 2.994 37.982 L 1.14 37.738 L 1.14 35.46 L 2.062 32.684 L 6.194 36.815 L 6.194 36.811 Z" fill="currentColor" stroke-width="1" />
-                    <path d="M 13.916 35.186 L 3.693 24.962 L 13.916 35.186 Z" stroke-width="1" />
-                    <path d="M 13.916 35.186 L 3.693 24.962" fill="none" stroke="currentColor" stroke-width="1" />
-                  </svg> icon to start customizing.</p>
+              <p class="wnp-body">Create a space that’s uniquely yours. Set the mood with Firefox wallpapers, custom colors, or your own photo. Open a new tab and click the
+                <svg class="wnp-inline-icon" width="16" height="16" viewBox="0 0 18 19" fill="currentColor" aria-label="pencil" role="img">
+                  <g clip-path="url(#a)"><path fill="currentColor" d="M16.935 4.622 14 1.672a1.035 1.035 0 0 0-1.46 0L2.135 12.062l-.95 4.1a1.03 1.03 0 0 0 1 1.25q.107.01.215 0l4.145-.95 10.39-10.38a1.035 1.035 0 0 0 0-1.46m-10.89 10.94-3.885.815.885-3.81 7.785-7.755 3 3zM14.5 7.087l-3-3 1.74-1.73 2.95 3z"/></g><defs><clipPath id="a"><path d="M0 .462h18v18H0z"/></clipPath></defs>
```

Comment:
I've cleaned up the icon to bare basics:

```suggestion
              <p class="wnp-body">Create a space that’s uniquely yours. Set the mood with Firefox wallpapers, custom colors, or your own photo. Open a new tab and click the
                <svg class="wnp-inline-icon" width="16" height="16" viewBox="0 0 16 16" fill="currentColor" aria-label="pencil" role="img">
                  <path fill="currentColor" d="M15.195 3.612 12.457.86a.965.965 0 0 0-1.362 0l-9.709 9.695-.887 3.825a.962.962 0 0 0 .933 1.167c.068.006.134.006.201 0l3.868-.887 9.694-9.685a.964.964 0 0 0 0-1.363M5.034 13.82l-3.625.761.826-3.555L9.499 3.79l2.799 2.799-7.264 7.231Zm7.889-7.908-2.799-2.799 1.624-1.614L14.5 4.298l-1.577 1.614Z"/>
```

Feel free to use this instead.